ai21_tokenizer/jurassic_tokenizer.py

```diff
@@ -19,11 +19,16 @@ class SpaceSymbol:
 class JurassicTokenizer(BaseTokenizer):
     def __init__(
         self,
-        model_path: PathLike,
+        model_path: Optional[PathLike] = None,
+        model_file_handle: Optional[BinaryIO] = None,
         config: Optional[Dict[str, Any]] = None,
     ):
+        JurassicTokenizer._assert_exactly_one(model_path, model_file_handle)
```

Comment:
use `self` here

Reply:
Yeah I wasn't sure what was better. Some online refs say `self` some say to use the class name